src/select/parts/styles.scss

```diff
@@ -88,8 +88,8 @@ $inlineLabel-border-radius: 2px;
     min-inline-size: max-content;
     block-size: 18px;
 
-    border-block: awsui.$border-width-field solid awsui.$color-border-item-selected;
-    border-inline: awsui.$border-width-field solid awsui.$color-border-item-selected;
+    border-block: awsui.$border-width-token solid awsui.$color-border-item-selected;
+    border-inline: awsui.$border-width-token solid awsui.$color-border-item-selected;
```

Comment:
Not for this line in particular, but for the select component

When select with inline label is rendered in table header, it still uses white background 

<img width="795" alt="image" src="https://github.com/user-attachments/assets/c0e8c9c9-2fd3-4d96-ad5e-70744cb9a0a8" />

Reply:
Will fix this. Great catch.